ext/readme.md

```diff
@@ -62,7 +62,7 @@ ext | `requests-oauthlib` | [1.2.0](https://pypi.org/project/requests-oauthlib/1
 ext | <code><b>six</b>.py</code> | [1.12.0](https://pypi.org/project/six/1.12.0/) | **`medusa`**, `adba`, `configobj`, `guessit`, `html5lib`, `imdbpie`, `Js2Py`, `knowit`, `python-dateutil`, `rebulk`, `subliminal`, `tvdbapiv2`, `validators` | -
 ext | `soupsieve` | [1.7](https://pypi.org/project/soupsieve/1.7/) | `beautifulsoup4` | -
 ext | `stevedore` | [1.30.1](https://pypi.org/project/stevedore/1.30.1/) | `subliminal` | -
-ext | `subliminal` | [6ac2fa2](https://github.com/Diaoul/subliminal/tree/6ac2fa23ee5baa7d8452552edaa7c4a8a00d237a) | **`medusa`** | -
+ext | `subliminal` | [develop@6ac2fa2](https://github.com/Diaoul/subliminal/tree/6ac2fa23ee5baa7d8452552edaa7c4a8a00d237a) | **`medusa`** | -
```

Comment:
This is for `mvt outdated`, see https://github.com/sharkykh/medusa_vendor_tools/issues/7:
>Iterate over all the packages in the lists (`{ext,lib}/readme.md`) and for each one check if they have a newer version.
>[...] I think I have to know the branch name in order to check for newer commits (for example subliminal has `master` and `develop`, we usually want `develop`).